assemblies/assembly-release-notes-fixed-security-issues.adoc

```diff
@@ -0,0 +1,10 @@
+:_content-type: ASSEMBLY
+[id="fixed-security-issues"]
+= Fixed security issues
+
+This section lists security issues fixed in {product} {product-version}.
+
+
+
+None.
```

Comment:
well, this isn't true, lol. But we don't have the massive list handy

Reply:
Oh. Still missing? Weird.

Reply:
captured in https://issues.redhat.com/browse/RHIDP-4290